### ISSUE TITLE
Label id uses random generated value and breaks snapshot tests

### ISSUE THREAD

**Issue by francocorreasosa** [auth0/cosmos#1267]
```diff
    @@ -264,11 +264,11 @@
            <div
              className="sc-iyvyFf bhhnDL"
            >
              <label
                className="sc-dqBHgY hqeZQu"
-               htmlFor="add_same-origin_header_to_authentication_endpoints-ubv6jvm3"
+               htmlFor="add_same-origin_header_to_authentication_endpoints-vymhlvxa"
              >
                Add Same-Origin header to authentication endpoints
              </label>
            </div>
            <div
```

**Comment by tehsis** [comment]
I have also reported this on: https://github.com/auth0/cosmos/issues/1268

**Comment by siddharthkp** [comment]
Fixed in #1374
